midterm/types/MidtermTabParamList.ts

```diff
@@ -0,0 +1,6 @@
+import { ScanStackParams } from "./ScanStackParams";
+
+export type MidtermTabParamList = {
+	Scan: ScanStackParams;
```

Comment:
This should stop TS from being angry within `Favorites.tsx`
```suggestion
    Scan: { screen: 'ProductDetails'; params: ScanStackParams['ProductDetails'] };
```